### ISSUE TITLE
Linux Support Locked to Steam Deck Only?

### ISSUE THREAD

**Issue by SolarNyan** [beebls/CSSLoader-Desktop#23]
I see that Windows has a standalone version of this, but is the Linux version only for Steam Deck's specific distribution of Linux? Decky Loader cannot be installed on a Non-Valve official HoloISO OS and cannot function as a backend for Linux, in my experience. I was trying to install this on my desktop Linux rig because the documentation lists Linux compatibility rather than just Steam Deck, and the AppImage loads and functions properly, but I am confused as to how the backend works.

If it's me misunderstanding the main page and the support is SD only, that's perfectly fine, of course, I was just curious as to how it all worked and if desktop Linux integration is at all possible at the moment, or at least, maybe considered for the future? I'd love to use it on my desktop rig to have theming parity with my Deck, but if not, that's okay, I appreciate the immense effort that's been put into the SD edition either way. Cheers!

**Comment by suchmememanyskill** [comment]
No, you should be able to run Decky Loader on linux desktop just fine. Do you have any error logs for that?

For windows, we have the standalone version because Decky is not available on windows. If you can run Decky, you should run CSS Loader through it

**Comment by SolarNyan** [comment]
Apologies for the late reply, kept forgetting to come back here and comment. >.<

I was able to to get it working! Turns out, Decky Loader's install was failing due to missing JQ. After installing that, the install script worked without a hitch! However, the CSSLoader appimage couldn't detect the backend, so I have to install the plugin inside of Big Picture through DL's plugin store and manage things from there.
